modules/adagioBidAdapter.js

```diff
@@ -38,7 +38,8 @@ const LOG_PREFIX = 'Adagio:';
 const FEATURES_VERSION = '1';
 export const ENDPOINT = 'https://mp.4dex.io/prebid';
 const SUPPORTED_MEDIA_TYPES = [BANNER, NATIVE, VIDEO];
-const ADAGIO_TAG_URL = 'https://script.4dex.io/localstore.js';
+const ADAGIO_TAG_URL = 'https://script.4dex.io/adagio.js';
```

Comment:
please lock this to a version and update what is available in https://github.com/Prebid-org/prebid-js-external-js-adagio-io/tree/master/dist

Reply:
Hi @patmmccann, we make regular updates, we can't lock to a version. We always work this way, is something changed? 

>  update what is available in… 
You mean, you want I do some cleanup?

Reply:
please see https://github.com/prebid/Prebid.js/issues/5804 ; as part of Prebid 5, Criteo locked their load to a version.